src/utils/useDownloadMedia.ts

```diff
@@ -18,11 +18,15 @@ export const useDownloadMedia = () => {
     if (where !== local.localname) {
       try {
         ipc?.createFolder(path.dirname(local.localname));
-        await ipc?.downloadFile(url, local.localname);
-        if (await ipc?.exists(local.localname)) {
-          if (safe) return await safeURL(local.localname);
-          return local.localname;
-        } else return url;
+        try {
+          await ipc?.downloadFile(url, local.localname);
+          if (await ipc?.exists(local.localname)) {
+            if (safe) return await safeURL(local.localname);
+            return local.localname;
+          } else return url;
+        } catch (e: any) {
+          return url;
+        }
       } catch (e: any) {
         return url;
       }
```

Comment:
I don't think we need a try catch inside of a try catch when we do the same thing with the catch.